.gitignore

```diff
@@ -10,9 +10,11 @@ xcuserdata
 # Fastlane
 fastlane/report.xml
 fastlane/Preview.html
-fastlane/screenshots
 fastlane/test_output
 
+fastlane/screenshots/*/*.png
+fastlane/screenshots/screenshots.html
```

Comment:
will you hate me if I ask for an alphabetically ordered .gitignore 🙊?

Reply:
Not at all 😄